README.md

```diff
@@ -120,11 +120,11 @@ expect(actual).to be_a_kind_of(expected)      # another alias
 
 ```ruby
 expect(actual).to be_truthy   # passes if actual is truthy (not nil or false)
-expect(actual).to be true     # passes if actual == true
+expect(actual).to be true     # passes if actual.equal?(true)
 expect(actual).to be_falsy    # passes if actual is falsy (nil or false)
-expect(actual).to be false    # passes if actual == false
-expect(actual).to be_nil      # passes if actual is nil
-expect(actual).to_not be_nil  # passes if actual is not nil
+expect(actual).to be false    # passes if actual.equal?(false)
+expect(actual).to be_nil      # passes if actual.nil?
+expect(actual).to_not be_nil  # passes if !actual.nil?
```

Comment:
Thanks for your review! 

In my understanding, it actually checks trurhy or falsy, instead of true or false.

https://github.com/rspec/rspec-expectations/blob/040d1b0bba83031c9ae432fd3a4462fdd1cc74e7/lib/rspec/matchers/built_in/be.rb#L68-L70

So below should be accurate one...?

```suggestion
expect(actual).to be_nil      # passes if actual.nil? returns truthy (not nil or false)
expect(actual).to_not be_nil  # passes if actual.nil? returns falsy (nil or false)
```

Reply:
The intent is that it respects Ruby semantics, which are that `nil?` is `false` unless it is `NilClass` where it returns `true` so the documentation should reflect that, if you'd like to make the check stricter I'm all for that.